lms/envs/common.py

```diff
@@ -1047,6 +1047,8 @@ def _make_mako_template_dirs(settings):
     'url_root': '/static/django-pyfs',
 }
 
+CERTIFICATE_DATE_FORMAT = "%B %-d, %Y"
```

Comment:
It will be good to add comments explaining the format supported here and also a comment explaining the constant.

Reply:
Added docs related to use of this new configuration.